server/bin/pbench-dispatch.sh

```diff
@@ -191,11 +195,29 @@ while read tbmd5; do
     popd > /dev/null 2>&4
     if [[ ${sts} -ne 0 ]]; then
         log_error "${TS}: Quarantined: ${tb} failed MD5 check" "${status}"
-        quarantine ${quarantine}/${controller} ${tb} ${tb}.md5
+        quarantine ${quarantine}/${controller} ${tbmd5} ${tb}
```

Comment:
I can't easily find the definition of the `quarantine` command...I hope the order of arguments doesn't matter....

Reply:
It's in `pbench-base.sh`, and it just takes a list of files.

Reply:
Changed them to use the same order.

Reply:
The two invocations in the new code use the same order as each other (which is good...), but it's a different order from the old code.  But, since you say it's just a list (that ordering doesn't matter), that's fine.